Source/WebCore/html/canvas/WebGL2RenderingContext.cpp

```diff
@@ -2497,29 +2497,29 @@ void WebGL2RenderingContext::deleteVertexArray(WebGLVertexArrayObject* arrayObje
 
     // validateWebGLObject generates an error if the object has already been
     // deleted, so we must replicate most of its checks here.
-    if (!arrayObject || isContextLost())
+    if (isContextLostOrPending() || !arrayObject)
         return;
 
     if (!arrayObject->validate(contextGroup(), *this)) {
-        synthesizeGLError(GraphicsContextGL::INVALID_OPERATION, "deleteVertexArray", "object does not belong to this context");
+        synthesizeGLError(GraphicsContextGL::INVALID_OPERATION, "delete", "object does not belong to this context");
```

Comment:
I suggest undoing this change. It's helpful to have the precise method name in the console logs.

Reply:
Out of all `delete*` methods, it's the only one that fully spells its name in case of invalid context error. I'll do a follow up to improve reporting throughout.